product_tax_multicompany_default/tests/test_product_tax_multicompany.py

```diff
@@ -131,3 +167,75 @@ def test_tax_not_default_set_match(self):
         product = product.sudo()
         self.assertIn(self.tax_10_cc2, product.taxes_id)
         self.assertIn(self.tax_10_sc2, product.supplier_taxes_id)
+
+    def test_set_multicompany_taxes(self):
```

Comment:
Why not decorating this with `@users` for testing with such user/s?

Reply:
Done